queries.sql

```diff
@@ -113,6 +113,69 @@ FROM species
 JOIN animals ON animals.species_id = species.id
 GROUP BY species.name;
 
+-- 1. Who was the last animal seen by William Tatcher?
+SELECT a.name AS animal_name
+FROM animals a
+JOIN visits v ON a.id = v.animal_id
+JOIN vets ve ON v.vet_id = ve.id
+WHERE ve.name = 'William Tatcher'
+ORDER BY v.date DESC
+LIMIT 1;
+-- 2. How many different animals did Stephanie Mendez see?
+SELECT COUNT(DISTINCT a.name) AS num_animals
+FROM animals a
+JOIN visits v ON a.id = v.animal_id
+JOIN vets ve ON v.vet_id = ve.id
+WHERE ve.name = 'Stephanie Mendez';
```

Comment:
- [ ] Stephanie Mendez has seen 4 different animals according to the information provided by the assignment, your query only returns 2 please double-check.